src/System.Private.Xml/src/System/Xml/Serialization/ReflectionXmlSerializationReader.cs

```diff
@@ -22,6 +23,8 @@ namespace Microsoft.XmlSerializer.Generator
 namespace System.Xml.Serialization
 #endif
 {
+    internal delegate void UnknownNodeAction(object o);
```

Comment:
Why did you use a custom delegate type instead of Action<object>? I'm not asking you change this as this works fine and can actually make the code slightly easier to read, just asking if there was a specific reason for create a custom delegate type.